### PR TITLE
Add January 2017 meeting

### PR DESCRIPTION
**Pull request by h-lame**

### CODE REVIEW

File: source/meetings/2017/january/index.html.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+_6:00pm_ to _8:00pm_ (talks start at _6:30pm_).  The venue, [Code
+Node](https://skillsmatter.com/locations/264-skills-matter-codenode) between
+Moorgate and Liverpool St. stations, is provided by [Skills
+Matter](http://www.skillsmatter.com).  [Full venue and registration details
+are given below](#jan17registration).
+
+Agenda
+------
+
+### A developers misadventures in business
```
Comment (tomstuart) [inline source/meetings/2017/january/index.html.md]: `s/developers/developer’s/`?

File: source/meetings/2017/january/index.html.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+[Najaf Ali](https://twitter.com/alinajaf) says:
+
+> I am developer who started a web application development consultancy. This
```
Comment (tomstuart) [inline source/meetings/2017/january/index.html.md]: a developer

**Comment by tomstuart** [comment]
comments aside, looks good to me! :shipit:

**Comment by h-lame** [comment]
Fair points.  Thanks for the review.

**Pull request merged**